src/docs/sdk/performance/dynamic-sampling-context.mdx

```diff
@@ -42,148 +34,140 @@ Multiple different traces can and should have different Dynamic Sampling Context
 
 ### SDKs
 
-SDKs are responsible for propagating **Dynamic Sampling Context** across all applications that are part of a trace.
+SDKs are responsible for propagating a **"Dynamic Sampling Context"** or **"DSC"** across all applications that are part of a trace.
 This involves:
 
-1. Collecting the information that makes up the DSC **or** extracting the DSC from incoming requests.
-2. Propagating DSC to downstream SDKs.
-3. Sending the DSC to Sentry via the `trace` envelope header.
+1. If there is an incoming request, extracting the DSC from incoming requests.
+2. If there is no incoming request, creating a new DSC by collecting the data that makes up the DSC.
+3. Propagating DSC to downstream SDKs (via the `baggage` header).
+4. Sending the DSC to Sentry (via the `trace` envelope header).
 
-Because there are quite a few things to keep in mind for DSC propagation and to avoid every SDK running into the same problems, we defined a [unified propagation mechanism](#unified-propagation-mechanism) (step-by-step instructions) that all SDK implementations should be able to follow.
+To align DSC propagation over all our SDKs, we defined a [unified propagation mechanism](#unified-propagation-mechanism) (step-by-step instructions) that all SDK implementations should be able to follow.
 
-#### Currently supported SDKs:
+## DSC Specification
 
-The currently supported SDKs are documented on the [Dynamic Sampling Product docs](https://docs.sentry.io/product/data-management-settings/server-side-sampling/getting-started/#current-limitations). Please update that list once you release an SDK that supports dynamic sampling features. To be considered a supported SDK for dynamic sampling, the SDK must support both creating the DSC at the head of the trace, and propagating the DSC from continuing traces (if possible for that environment).
+All of the attributes in the table below are required (non-optional) in a sense, that when they are known to an SDK at the time an envelope with an event (transaction or error) is sent to Sentry, or at the time a baggage header is propagated, they must also be included in said envelope or baggage. In any case, `trace_id`, `public_key`, and `sample_rate` should always be known to an SDK, so these values are strictly required.
```

Comment:
Can we clarify required and strictly required please.
Certain values are required only in certain context.

Reply:
added a sentence below explaining it.